app/src/services/transformService.js

```diff
@@ -40,6 +40,11 @@ const transformService = {
       const ipcPlan = {...obj.ipcPlan};
 
       const location = {...obj.location};
+
+      // split city lookup result into city and province
+      location.city = obj.location.city.substring(0, obj.location.city.lastIndexOf(', '));
+      location.province = obj.location.city.substring(obj.location.city.lastIndexOf(', ') + 2);
```

Comment:
This logic could be unstable as it has a lot of assumptions on how something is structured. Instead of using substrings, you can do something more succinct with the `str.split()` function and then `str.trim()` off the whitespace around it. For example...
```js
"one two, bc".split(',').map(x => x.trim());
```
Yields `[ 'one two', 'bc' ]`. You can assign to the correct variables easily from there.

Reply:
We have to be careful here.  Localities have their own comma (ex. Victor Rd, Mayne Island,  BC), so splitting on comma isn't a great idea.  But even going with lastIndex, we don't know that city has come from the lookup.  Users can enter whatever they want.  So, let's try to parse off ", BC" and get city/province, but if that fails, it all goes into city.  Also, now that I look at it, let's do nothing.  In Read only mode/word doc, you lose the ", BC".

Reply:
yes. good call!

Reply:
It's a fair point - there's too many possibilities for format - keeping what the user entered w/o transform is safer at this point.